test/rpc/test_data/get_events.py

```diff
@@ -0,0 +1,119 @@
+"""
+RPC get events test data.
+"""
+
+from test.shared import PREDEPLOY_ACCOUNT_CLI_ARGS
+
+BLOCK_0_LATEST = {
+    "from_block": "0",
+    "to_block": "latest",
+}
+
+BLOCK_0_3 = {
+    "from_block": "0",
+    "to_block": "3",
+}
+
+BLOCK_3_4 = {
+    "from_block": "3",
+    "to_block": "4",
+}
+
+BLOCK_0_LATEST_CHUNK_3_0 = {
+    "from_block": "0",
+    "to_block": "latest",
+    "chunk_size": 3,
+}
+
+BLOCK_0_LATEST_CHUNK_3_1 = {
+    "from_block": "0",
+    "to_block": "latest",
+    "chunk_size": 3,
+    "continuation_token": "1",
+}
+
+BLOCK_0_LATEST_ADDRESS = {
+    "from_block": "0",
+    "to_block": "latest",
+    "address": "0x62230eA046a9a5fbc261ac77d03c8d41e5d442db2284587570ab46455fd2488",
+}
+
+BLOCK_0_LATEST_KEY = {
+    "from_block": "0",
+    "to_block": "latest",
+    "keys": ["0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"],
+}
+
+BLOCK_0_LATEST_KEYS = {
+    "from_block": "0",
+    "to_block": "latest",
+    "keys": [
+        "0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9",
+        "0x3db3da4221c078e78bd987e54e1cc24570d89a7002cefa33e548d6c72c73f9d",
+    ],
+}
+
+BLOCK_0_LATEST_ADDRESS_KEYS = {
+    "from_block": "0",
+    "to_block": "latest",
+    "address": "0x62230eA046a9a5fbc261ac77d03c8d41e5d442db2284587570ab46455fd2488",
+    "keys": [
+        "0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9",
+        "0x3db3da4221c078e78bd987e54e1cc24570d89a7002cefa33e548d6c72c73f9d",
+    ],
+}
+
+EVENT_DATA_0 = "[0, 0]"
+EVENT_DATA_1 = "[0, 1]"
+EVENT_FEE = (
+    "[1483697464188569092488551792700745002099692778312526338015109648680562628122, "
```

Comment:
This looks very ugly, the formatter probably doesn't touch it because it's a single multiline string. Perhaps you could do call `str(...)` on an actual object defined with `[...]`.

Reply:
Fixed.